File: services/auth/src/main/java/com/example/iter/auth/dto/request/UserDeleteRequest.java
package com.example.iter.auth.dto.request;

import jakarta.validation.constraints.Size;

public record UserDeleteRequest(
        @Size(min = 8, max = 32, message = "비밀번호는 8자 이상 32자 이하여야 합니다.")
        String password
) {
}

File: services/auth/src/main/kotlin/com/example/iter/auth/dto/request/UserDeleteRequest.kt
package com.example.iter.auth.dto.request

import jakarta.validation.constraints.Size

@JvmRecord
data class UserDeleteRequest(
    @field:Size(min = 8, max = 32, message = "비밀번호는 8자 이상 32자 이하여야 합니다.")
    val password: String?,
)
